src/renderers/shaders/ShaderChunk/bsdfs.glsl

```diff
@@ -1,11 +1,19 @@
-float calcLightAttenuation( const in float lightDistance, const in float cutoffDistance, const in float decayExponent ) {
+// convert from intensity (power / solid angle) to irradiance assuming a spherical light falloff
+// this is only valid for spot and point lights
+// physically correct inverse quadratic falloff is achieved when decayExponent is 2.
+// discussed here: https://github.com/mrdoob/three.js/issues/7608
+float intensityToIrradianceSphericalFalloffCoefficient( const in float lightDistance, const in float cutoffDistance, const in float decayExponent ) {
 
 	if ( decayExponent > 0.0 ) {
 
-	  return pow( saturate( -lightDistance / cutoffDistance + 1.0 ), decayExponent );
+		float distanceFalloff = 1.0 / pow( max( lightDistance, EPSILON ), decayExponent );
```

Comment:
I think generally people are okay standardizing on SI units: https://github.com/mrdoob/three.js/issues/6259  If we start allowing for picking units here, it is hard to argue why it should be contained to just this situation and then eventually it could infect every single parameter in ThreeJS that could have units specifications that deviate from SI (intensity on lights, time in animations, emissive color on materials, lightMap on materials, etc.)  I would suggest we stick to a single unit for each quantity that is as standard as possible.